docs/assignment1.md

```diff
@@ -1,62 +1,154 @@
+</tr>
+<tr>
+<td>F8</td>
+<td>Inventory</td>
+<td>The game shall have a simple <b>inventory system</b> using slots for weapons, currency and other items. Simple means there is no weight and volume system.
+Weapons are not stackable, while currency and other items are.</td>
+</tr>
+<tr>
+<td>F9</td>
+<td>Time Mechanic</td>
+<td>The game shall have a <b>time system</b> for story-telling purposes. Time will be implemented with a turn-based system, for example: every <i>x</i> turns
+<i>y</i> amount of time passes. Some actions will pass time immediately, such as sleeping.</td>
+</tr>
+</table>
 
 ### Quality requirements
-Author(s): `name of the team member(s) responsible for this section`
-
-As a preamble to the table, you can discuss the main line of reasoning you followed for coming up with the quality requirements listed below.
+Author(s): Claudia Grigoras
 
 | ID  | Short name  | Quality attribute | Description  |
 |---|---|---|---|
-| QR1  | Commands sanity checks | Reliability  | When the player issues a command, the syntax of the command shall always get validated against the format specified in F2 |
-| QR2  | Extensible world | Maintainability  | The video game shall be easilty extendable in terms of levels, worlds, interaction points  |
-| QR3  | Instantaneous results | Responsiveness  | Once the scores of all soccer players are provided by the user, the results of the virtual matches shall be available within 1 second |
-| QR4  | ... | ... | ... |
+| QR1  | Commands sanity checks | Reliability | When the player issues a command, the syntax of the command shall always get validated against the format specified in F3 |
+| QR2  | Adaptable system | Maintainability & Responsiveness | The video game shall be easily adaptable to new stories and worlds. The system should not be dependent to one story only (when changing the story or the world, only variables and actions should change) |
+| QR3  | Loading game state | Responsiveness | Once the player types in a command, loading the game state shall be executed in less than 5 seconds regardless of the computer capabilities |
+| QR4  | Accessing the commands | Usability & Availability | The players shall at any point within the game be able to access the available commands at that particular level in game (through the command specified at F3 – examine\) |
+| QR5  | Secured connections | Security | When playing in a multiplayer mode, the system should provide a safe network connection to both user  |
+| QR6  | Real-time commands multiplayer mode | Responsiveness & Reliability | The system shall have a maximum latency of 2 seconds for providing commands and loading game state between 2 players in multiplayer mode |
+| QR7  | Un-biased game | Reliability & Responsiveness | The system shall ensure that the statistics provided at F6 are correct and un-biased and that the combat mentioned at F7 is unbiased and always gives a 50/50 chance roll. |
```

Comment:
On the first part related to F6, did you mean balanced numerical values with relation to some predetermined difficulty level? 
Responsiveness is not really a factor here either. I would suggest usability.